pkg/controllers/node/termination/controller.go

```diff
@@ -133,7 +133,8 @@ func (c *Controller) removeFinalizer(ctx context.Context, n *v1.Node) error {
 			return client.IgnoreNotFound(fmt.Errorf("patching node, %w", err))
 		}
 		metrics.NodesTerminatedCounter.With(prometheus.Labels{
-			metrics.NodePoolLabel: n.Labels[v1beta1.NodePoolLabelKey],
+			metrics.NodePoolLabel:     n.Labels[v1beta1.NodePoolLabelKey],
```

Comment:
Only added it for NodesTermianted for completeness, if we add it to NodesCreated